packages/client/src/helpers/commonQueryVariables.ts

```diff
@@ -0,0 +1,41 @@
+import { LensContext } from '../context';
+import { ImageTransform } from '../graphql';
+import {
+  ProfileStatsArg,
+  ProfileStatsCountOpenActionArgs,
+  PublicationOperationsActedArgs,
+  PublicationStatsCountOpenActionArgs,
+  PublicationStatsInput,
+  RateRequest,
+} from '../graphql/types.generated';
+import { AppId } from '../queryParams';
+
+type CommonQueryVariables = {
+  publicationImageTransform?: ImageTransform;
+  publicationOperationsActedArgs?: PublicationOperationsActedArgs;
+  publicationStatsInput?: PublicationStatsInput;
+  publicationStatsCountOpenActionArgs?: PublicationStatsCountOpenActionArgs;
+  profileCoverTransform?: ImageTransform;
+  profilePictureTransform?: ImageTransform;
+  profileStatsArg?: ProfileStatsArg;
+  profileStatsCountOpenActionArgs?: ProfileStatsCountOpenActionArgs;
+  profileMetadataSource?: AppId;
+  rateRequest?: RateRequest;
+};
+
+export function commonQueryVariables({ params }: LensContext): CommonQueryVariables {
+  return {
+    publicationImageTransform: params.image?.medium,
+    publicationOperationsActedArgs: {},
+    publicationStatsInput: { metadata: { publishedOn: params.statsFor } },
+    publicationStatsCountOpenActionArgs: {},
+    profileCoverTransform: params.profile?.cover,
+    profilePictureTransform: params.profile?.thumbnail,
+    profileStatsArg: { forApps: params.statsFor },
+    profileStatsCountOpenActionArgs: {},
```

Comment:
some of them are still not possible to configure but at least all are defined in this single place